presto-product-tests/README.md

```diff
@@ -107,16 +107,32 @@ following command:
 
 ```
 ./mvnw install -DskipTests
-presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query
+presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query,mysql_connector,postgresql_connector
 ```
 
-where `<profile>` is one of either:
+where [profile](#profile) is one of either:
 - **multinode** - pseudo-distributed Hadoop installation running on a
  single Docker container and a distributed Presto installation running on
- multiple Docker containers.
+ multiple Docker containers. For multinode the default configuration is 1 coordinator and 1 worker.
 - **singlenode** - pseudo-distributed Hadoop installation running on a
  single Docker container and a single node installation of Presto also running
- on a single Docker container.
+ on a single Docker container. With no impersonation presto accesses hdfs as the user mentioned in -DHADOOP_USER_NAME.
+ The data files for the tables created in hive will belong to user specified by -DHADOOP_USER_NAME in jvm.config.
+- **singlenode-hdfs-impersonation** - pseudo-distributed Hadoop installation running on a
+ single Docker container and a single node installation of Presto also running
+ on a single Docker container. Presto impersonates the user who is running the query when accessing HDFS.
+ The user specified by -DHADOOP_USER_NAME in jvm.config must be allowed to impersonate this user.
+- **singlenode-kerberos-hdfs-impersonation** - pseudo-distributed kerberized Hadoop installation running on a
+ single Docker container and a single node installation of kerberized Presto also running
+ on a single Docker container. This profile has kerberos impersonation. Presto impersonates the user who
+ is running the query when accessing HDFS. The principal specified by the hive.hdfs.presto.principal property
```

Comment:
I don't think that we need to add `The principal specified by the hive.hdfs.presto.principal property in hive.properties file must be allowed to impersonate this user` explanation here. It is explained in the hive connector documentation. Here we are the ones who provide the docker images for the environments. So, we must ensure that specified principal can impersonate. So, i would rather remove this sentence.